tests/foreman/ui/test_ldap_auth.py

```diff
@@ -0,0 +1,385 @@
+        functional areas in UI
+
+        @status: Manual
+
+        """
+
+    @stubbed()
+    def test_ldap_auth_ad_user_disabled(self):
+        """@test: LDAP - AD user activity when AD user account has been deleted
+        or deactivated
+
+        @feature: LDAP Authentication
+
+        @steps:
+        1.  Login to the foreman UI
+        2. Delete or disable userid on AD server side
+
+        @assert: This is handled gracefully (user is logged out perhaps?)
+        and no data corruption
+
+        @status: Manual
+
+        """
+
+    @stubbed()
+    def test_ldap_auth_openldap_basic_no_roles(self):
+        """@test: Login with LDAP - RHDSLDAP that has no roles / rights
+
+        @feature: LDAP Authentication
+
+        @setup: assure properly functioning RHDSLDAP server for authentication
+
+        @steps:
+        1. Login to server with a RHDSLDAP user
+
+        @assert: Log in to foreman UI successfully but has no access to
+        functional areas of UI.
+
+        @status: Manual
+
+        """
+
+    @stubbed()
+    def test_ldap_auth_rhdsldap_basic_roles(self):
+        """@test: Login with LDAP - RHDS LDAP for user with roles/rights
+        assigned.
+
+        @feature: LDAP Authentication
+
+        @setup: assure properly functioning RHDS LDAP server for authentication
+
+        @steps:
+        1. Login to server with a RHDS LDAP id
+
+        @assert: Log in to foreman UI successfully and can access appropriate
+        functional areas in UI
+
+        @status: Manual
+
+        """
+
+    @stubbed()
+    def test_ldap_auth_rhdsldap_user_disabled(self):
+        """@test: LDAP - RHDSLDAP user activity when RHDS ldap account has been
+        deleted or deactivated
+
+        @feature: LDAP Authentication
+
+        @steps:
+        1. Login to the foreman UI
+        2. Delete or disable userid on RHDS LDAP server side
+
+        @assert: This is handled gracefully (user is logged out perhaps?) and
+        no data corruption
+
+        @status: Manual
+
+        """
+
+    @stubbed()
+    def test_ldap_auth_multiple_ldap_backends(self):
+        """@test: LDAP - multiple LDAP servers kafo instance
+
+        @feature: LDAP Authentication
+
+        @setup: Assure more than one ldap server backend is provided for
+        sat6
+
+        @steps:
+        1. Attempt to login with a user that exists on one ldap server
+        2. Logout and attempt to login with a user that exists on other ldap
+        server(s)
+
+        @assert: Log in to foreman UI successfully for users on both LDAP
+        servers.
+
+        @status: Manual
+
+        """
+
+    @stubbed()
+    def test_ldap_auth_multiple_ldap_namespace_collision(self):
+        # devnote:
+        # users have auth_source which could distinguish them, but validation
+        # would fail atm
+        """@test: LDAP - multiple LDAP servers colliding namespace
+        (i.e "jsmith")
```

Comment:
ok... now i am just being super picky! But you should use "e.g." here instead of "i.e."